src/plugin-sidebar.js

```diff
@@ -155,6 +158,17 @@ const CreateBlockThemePlugin = () => {
 										'create-block-theme'
 									) }
 								</Button>
+								<Button
+									icon={ code }
+									onClick={ () =>
+										setIsGlobalStylesEditorOpen( true )
+									}
+								>
+									{ __(
+										'View Global Styles Customizations',
```

Comment:
It looks a weird I think because of the length of the text. 

![Screenshot 2024-08-12 at 11 29 18 AM](https://github.com/user-attachments/assets/c45cdecf-3421-46ad-9680-9071740028b3)
```suggestion
										'View Custom Styles',
```

Or `View Style Customizations`?